pkg/policy/distillery_test.go

```diff
@@ -424,26 +420,24 @@ func Test_MergeRules(t *testing.T) {
 		{3, api.Rules{rule____NoAllow, rule____NoAllow, rule____NoAllow, rule__L4__Allow, ruleL3____Allow}, MapState{mapKeyAllow___L4: mapEntryL7None_, mapKeyAllowFoo__: mapEntryL7None_}},
 		{4, api.Rules{rule____NoAllow, rule____NoAllow, ruleL3L4__Allow, rule____NoAllow, rule____NoAllow}, MapState{mapKeyAllowFooL4: mapEntryL7None_}},
 		{5, api.Rules{rule____NoAllow, rule____NoAllow, ruleL3L4__Allow, rule____NoAllow, ruleL3____Allow}, MapState{mapKeyAllowFooL4: mapEntryL7None_, mapKeyAllowFoo__: mapEntryL7None_}},
-		{6, api.Rules{rule____NoAllow, rule____NoAllow, ruleL3L4__Allow, rule__L4__Allow, rule____NoAllow}, MapState{mapKeyAllow___L4: mapEntryL7None_}},                                    // Differs from spreadsheet(!)
+		{6, api.Rules{rule____NoAllow, rule____NoAllow, ruleL3L4__Allow, rule__L4__Allow, rule____NoAllow}, MapState{mapKeyAllowFooL4: mapEntryL7None_, mapKeyAllow___L4: mapEntryL7None_}}, // Differs from spreadsheet(!)??
```

Comment:
Checked the changed lines against the spreadsheet. The cases 12-15 all have an L4-only rule redirecting to proxy, and an L3/L4 rule that is allowed without redirecting. Now the latter case will not be sent to the proxy as it is not required for policy enforcement, but this may be problematic for a visibility policy?

Reply:
Updated the comments to reflect the above!